scripts/resolve_bugs.pl

```diff
@@ -42,8 +44,15 @@
 $auto_user->{bless_groups} = [ Bugzilla::Group->get_all ];
 Bugzilla->set_user($auto_user);
 
-my $query = { resolution => '---', product => $product };
-$query->{component} = $component if defined $component;
+if ($buglist) {
+    my $uri = URI->new($buglist);
+    die("Invalid input") if $uri->host eq 'bugzilla.mozilla.org';
```

Comment:
I think this is a logic error - `eq` should be `ne` here, right?

I'd technically like to see 'bugzilla.mozilla.org' loaded from a Bugzilla::Config or something like that, rather than hard-coded, but I understand the expedience of doing it this way.